modules/calib3d/test/test_undistort.cpp

```diff
@@ -1807,4 +1807,79 @@ TEST(Calib3d_initUndistortRectifyMap, regression_14467)
     EXPECT_LE(cvtest::norm(dst, mesh_uv, NORM_INF), 1e-3);
 }
 
+TEST(Calib3d_initInverseRectificationMap, regression_14467)
```

Comment:
> regression_14467

Please change this name

Reply:
fixed. Assumed number is arbitrary and changed it to `regression_14468`.